### DISCUSSION TITLE
Prioritize my repos higher in Favorites search

### DISCUSSION THREAD

**Discussion by KennethWussmann** [community/community#3578] [Mobile]
Hey,
I love the Favorites list, because I can select repos I often want to check on my mobile phone. Repos of my organizations or my own, for example.

But it's often not very easy to add repos to this list. If I search for `website` to find my repo called `website`, I get a list full with people that have called their repos `website` as well, but to find mine I often have to search quite a bit. 
It would be way easier if repos I am involved with, would show up on top of the list or let it be two tabs of search results. Like repos called the same, are tagged with this keyword or exist in organizations I am member of.

It would already be a great improvement if you could add repos to this list on the page of the repo. Like in the three dot menu where you can report or share, a quick option to add / remove this repo to my favorites.

**Comment by candyho** [comment]
Hey thanks for your feedback. I've added your request to a tracking issue in our internal repo. Hopefully we'll have an update to share soon.
